src/@chakra-ui/gatsby-plugin/components/Avatar.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  createMultiStyleConfigHelpers,
+  cssVar,
+  defineStyle,
+} from "@chakra-ui/react"
+import { avatarAnatomy } from "@chakra-ui/anatomy"
+import { avatarDefaultTheme, defineMergeStyles } from "./components.utils"
+import { omit, pick } from "lodash"
+
+const { defineMultiStyleConfig, definePartsStyle } =
+  createMultiStyleConfigHelpers(avatarAnatomy.keys)
+
+const { baseStyle: defaultBaseStyle, sizes: defaultSizes } = avatarDefaultTheme
+
+const $border = cssVar("avatar-border-color", "transparent")
+const $mlBySize = cssVar("ml-by-size")
+
+const baseStyleContainer = defineStyle((props) =>
+  defineMergeStyles(defaultBaseStyle?.(props).container, {
+    [$border.variable]: "transparent",
+    borderWidth: "1px",
+    "&:hover, [data-peer]:hover ~ &": {
+      boxShadow: "buttonHover",
```

Comment:
Can't remember where we ended up with this but talking async with @nloureiro, we said that it would be nice if we can use a more dynamic shadow size for the Avatar.

```tsx
// we tested with 0.15em and its fine
boxShadow="0.15em 0.15em 0 color"
```
but I think this will only apply for the Avatar, the other components should keep using the `primary` shadow.

Let me know what do you think on this implementation @TylerAPfledderer

Reply:
@pettinarip I've pushed this change for you to view.

Though it looks very minor. I'm seeing clipping which does not occur if pixel values are applied. 
![image](https://github.com/ethereum/ethereum-org-website/assets/65234762/4a6df34a-68d8-4667-bbbd-e5162c97a359)

Not sure what is causing this on my end.